staff/RominaFior/blog/index.html

```diff
@@ -0,0 +1,123 @@
+                    </p>
+                </header>
+            </article>
+            <article id="entradas">
+
+                <header>
+                    <h3>Medio Ambiente</h3>
+                    <h4>11/04/2018 RominaFior Planeta</h4>
+                    <img src="images/planet.png" alt="planeta" width="250">
+                    <p> Como hemos visto, el usar animales para alimentarnos hace que también tengamos que producir los vegetales
+                        para mantenerlos. Como además se necesita más cantidad de vegetales para alimentarlos que la que
+                        consumiríamos nosotros directamente, el desperdicio de comida, degradación del terreno, contaminación,
+                        etc es mucho mayor. Hay que tener en cuenta, también, que los animales de por sí son mucho más contaminantes
+                        que los vegetales, por lo que parece claro que el ser vegano es muy beneficioso para el medio ambiente
+                        y, por lo tanto, para todos los animales.
+                    </p>
+                </header>
+            </article>
+            <article>
+                <header>
+
+                    <h3>Filosofía</h3>
+                    <h4>11/04/2018 RominaFior Naturaleza</h4>
+                    <img src="images/design-3289964_640.png" alt="naturaleza" width="250">
+                    <p>El veganismo es un estilo de vida basado en el respeto hacia los animales. Considerar que los animales
+                        son individuos con intereses que merecen ser respetados tiene implicaciones prácticas que evitan
+                        que miles de ellos sufran y/o mueran cada segundo en mataderos, centros de experimentación, zoos,
+                        acuarios, circos, granjas peleteras, etc.
+                    </p>
+                </header>
+            </article>
+        </section>
+        <section>
+            <a id="comentarios">
+                <h4>COMENTARIOS</h4>
+            </a>
+            <textarea name="comentarios" id="comentarios" cols="30" rows="10"></textarea>
+        </section>
+    </main>
+    <aside>
+        BUSCAR ESTE BLOG
+        <br>
+        <input type="text" value="" placeholder="">
+        <button>Buscar</button>
+        <br> SEGUIDORES
+        <br>
+        <input type="text" value="" placeholder="">
+        <button>Seguir</button>
+        <br> SUSCRIBIRSE POR E-MAIL
+        <br>
+        <input type="text" value="" placeholder="Escribe tu e-mail">
+        <button>Enviar</button>
+        <br> ARCHIVO BLOG
+        <ul>
+            <li>2015</li>
+            <ul><li>Medicina</li></ul>
+            <li>2016</li>
+            <ul><li>Cosmética</li></ul>
+            <li>2017</li>
+            <ul><li>Yoga</li></ul>
+            <li>2018</li>
+            <ul><li>Restaurantes veganos Barcelona</li></ul>
+        </ul>
+
+    </aside>
+    <footer>
+        <p>© 2018 ROMINA FIOR ALL RIGHTS RESERVED</p>
```

Comment:
WARN! for special symbols, like copyright, use always html entities. in this case: 

```&copy;```

SEE https://www.w3schools.com/html/html_entities.asp for more details

Reply:
done!